CONTRIBUTING.md

```diff
@@ -41,6 +41,7 @@ The test suite requires that you have the following installed:
 
 * [libmagic]
 * [SQLite]
+* [libvips and libvips-dev] - please download the appropriate package suiting your operating system.
```

Comment:
Do you have to download both or just one depending on the OS? If latter, then "or" is more correct than "and".

I think it's better to simplify it to only "libvips", the most common one, and leave it up to the dev for their specific OS. Any thoughts?

Reply:
Sure, that makes sense. The tests will work just fine without  libvips-dev. I'll remove and update the link straight to the "how to install my build" page.